src/Illuminate/Cache/composer.json

```diff
@@ -21,7 +21,7 @@
         "illuminate/support": "^9.0"
     },
     "provide": {
-        "psr/simple-cache-implementation": "1.0"
+        "psr/simple-cache-implementation": "1.0|2.0|3.0"
```

Comment:
I'm not sure that syntax is supported? Don't we have to specify "the version" we provide? What do symfony do here?

Reply:
It is: https://github.com/symfony/cache/blob/5.4/composer.json#L20